openfe_gromacs/tests/protocols/test_create_systems.py

```diff
@@ -0,0 +1,97 @@
+# This code is part of OpenFE and is licensed under the MIT license.
+# For details, see https://github.com/OpenFreeEnergy/openfe-gromacs
+import gufe
+import numpy as np
+from openmm import NonbondedForce
+from openmm.app import GromacsGroFile, GromacsTopFile
+
+from openfe_gromacs.protocols.gromacs_md.md_methods import GromacsMDProtocol
+from openfe_gromacs.protocols.gromacs_utils import create_systems
+
+
+def test_interchange_gromacs(alanine_dipeptide_component, tmpdir):
```

Comment:
Could you open an issue that effectively says something like adding a single point energy test - interchange should already do this but it looks like folks want us to verify.